ee/backend/app/src/main/kotlin/io/tolgee/ee/data/task/CalculateScopeRequest.kt

```diff
@@ -0,0 +1,13 @@
+package io.tolgee.ee.data.task
+
+import io.tolgee.model.enums.TaskType
+import jakarta.validation.constraints.NotNull
+
+data class CalculateScopeRequest(
+  @field:NotNull
+  var language: Long,
+  @field:NotNull
+  var type: TaskType,
+  @field:NotNull
+  var keys: MutableSet<Long>? = null,
```

Comment:
here we can use emptyMutbaleSet() as default value, so we can avoid the not-null forcing